examples/basics/parameters/get-tabledata.js

```diff
@@ -0,0 +1,71 @@
+const createSession = require('../../session');
+
+// inline script with a simple table
+const script = `
+exampletable:
+load * Inline [
+dim, val
+foo, 23
+bar, 42
+c, 54
+d, 21
+];
+`;
+
+const session = createSession();
+
+/**
+ * Write the tabelContent to console.
+ * @param {*} tabelContent
+ */
+function print(tableContent) {
+  for (let i = 0; i < tableContent.length; i += 1) {
+    console.log(`dim: ${tableContent[i].qValue[0].qText} val: ${tableContent[i].qValue[1].qText}`);
+  }
+}
+
+// local variable to store the generated sessionapp
+let sessionApp;
+
+session.open()
+  .then((global) => {
+    console.log('Session opend');
+    return global.createSessionApp();
+  })
+  .then((app) => {
+    console.log('App generated');
+    sessionApp = app;
+    return app.setScript(script);
+  })
+  .then(() => {
+    console.log('Script recieved');
+    return sessionApp.doReload();
+  })
+  .then(() => {
+    const returnProm = Promise.all([
+      sessionApp.getTableData(0, 4, false, 'exampletable')
```

Comment:
This is the core of the example, please add comments explaining the difference between the two calls and how parameters are passed.